src/components/searchbar/index.js

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+import React, { useState } from "react";
+import Results from "../results";
+import "./search.css";
+
+const SearchBar = () => {
+  const [searchInput, setSearchInput] = useState("");
+  const [repos, setRepos] = useState([]);
+
```

Comment:
Nice use of deconstructing to set the state, can you give more information on how it works?